study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/WorkflowBuilder.java

```diff
@@ -29,13 +29,23 @@ public WorkflowBuilder(Config cfg, StudyDto studyDto) {
     }
 
     void run(Handle handle) {
-        insertTransitions(handle);
+        insertWorkflow(handle);
     }
 
-    private void insertTransitions(Handle handle) {
+    void runUpdate(Handle handle) {
         if (!cfg.hasPath("workflowTransitions")) {
             return;
         }
+        int deletionCount = handle.attach(WorkflowDao.class).deleteStudyWorkflow(studyDto.getId());
+        LOG.info("Deleted {} workflow transitions PEX expressions", deletionCount);
```

Comment:
The count we get back is actually for the number of transitions, so the log message here is a bit misleading.

Reply:
RIght. 6398589b